packages/backups-cli/src/utils/env.ts

```diff
@@ -53,42 +51,46 @@ export class Environment {
 
   private static async loadFromVault(
     environment: EnvironmentType,
-    name: string
+    name: string,
+    separateVault: boolean
   ): Promise<string | null> {
     const spinner = ora()
 
-    spinner.start(`Please wait, obtaining key for ${environment} from Vault...`)
+    const projectName = `${process.env.PROJECT_NAME}${
+      separateVault ? '-backups' : ''
```

Comment:
Instead of using a hardcoded suffix I would rather change the compose file. It should accept a list of vault projects.

Later you should iterate through that list and assign variables if those are not defined yet in previous iteration.

This way it will be more flexible. For example we can add more that two vault projects if needed.